### ISSUE TITLE
ChainedClasses error undefined method `members' for "\n":String

### ISSUE THREAD

**Issue by lencioni** [brigade/scss-lint#643]
I am seeing the following error when enabling the ChainedClasses rule

```
SCSSLint::Linter::ChainedClasses raised unexpected error linting file path/to/file.scss: 'undefined method `members' for "\n":String'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/linter/chained_classes.rb:18:in `chained_class?'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/linter/chained_classes.rb:8:in `block in visit_sequence'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/linter/chained_classes.rb:7:in `each'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/linter/chained_classes.rb:7:in `visit_sequence'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/selector_visitor.rb:12:in `visit_selector_node'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/selector_visitor.rb:5:in `visit_selector'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/selector_visitor.rb:21:in `block in visit_members'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/selector_visitor.rb:20:in `each'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/selector_visitor.rb:20:in `visit_members'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/selector_visitor.rb:14:in `visit_selector_node'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/selector_visitor.rb:5:in `visit_selector'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/linter.rb:133:in `visit'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/linter.rb:148:in `block in visit_children'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/linter.rb:146:in `each'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/linter.rb:146:in `visit_children'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/sass-3.4.19/lib/sass/tree/visitors/base.rb:38:in `visit'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/linter.rb:137:in `visit'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/linter.rb:148:in `block in visit_children'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/linter.rb:146:in `each'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/linter.rb:146:in `visit_children'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/sass-3.4.19/lib/sass/tree/visitors/base.rb:38:in `visit'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/linter.rb:137:in `visit'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/linter.rb:41:in `run'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/runner.rb:51:in `run_linter'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/runner.rb:33:in `block in find_lints'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/runner.rb:31:in `each'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/runner.rb:31:in `find_lints'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/runner.rb:19:in `block in run'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/runner.rb:18:in `each'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/runner.rb:18:in `run'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/cli.rb:65:in `scan_for_lints'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/cli.rb:50:in `act_on_options'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/lib/scss_lint/cli.rb:27:in `run'
/Users/joe/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/scss_lint-0.43.1/bin/scss-lint:6:in `<top (required)>'
/Users/joe/.rbenv/versions/2.2.3/bin/scss-lint:23:in `load'
/Users/joe/.rbenv/versions/2.2.3/bin/scss-lint:23:in `<main>'
Report this bug at https://github.com/brigade/scss-lint/issues
```

with the following scss

```scss
.myclass {
  opacity: .6;

  &:hover,
  &:focus {
    opacity: 1;
  }
}
```
